fvm/bootstrap.go

```diff
@@ -55,13 +56,28 @@ var DefaultMinimumStorageReservation = func() cadence.UFix64 {
 	return value
 }()
 
+var DefaultTransactionFees = func() cadence.UFix64 {
```

Comment:
This doesn't seem to be used. I guess it's here and exported so it can be used by other code. 
Given that the default is 0 (`transactionFee` has its Go default 0), this might get used the wrong way.

Maybe initialize to 0.0 below?

Reply:
Yes this is only exported to be used elsewhere. I will explicitly init them to 0.0 here.